app/services/taxonomy/publish_taxon.rb

```diff
@@ -25,6 +25,14 @@ def publish
       raise(InvalidTaxonError, I18n.t('errors.invalid_taxon'))
     end
 
+    def unpublish(type:, alternative_path: nil)
```

Comment:
If this is a one-off, perhaps just use `publishing_api.unpublish` directly, like in the controller: 

https://github.com/alphagov/content-tagger/blob/master/app/controllers/taxons_controller.rb#L82